cz_version_bump/thanks.py

```diff
@@ -53,12 +53,16 @@ def thanks_message(self, commit: git.GitCommit) -> str:
 
     def third_party_contributors(self: Thanker, commit: git.GitCommit) -> Iterable[str]:
         for contributor in self.contributors(commit):
-            if contributor not in self.org_members:
-                yield contributor
+            # Exclude org members and dependabot from the thanks message
+            if contributor.login not in self.org_members and contributor.type != "Bot":
+                yield contributor.login
 
-    def contributors(self: Thanker, commit: git.GitCommit) -> Iterable[str]:
+    def contributors(
+        self: Thanker,
+        commit: git.GitCommit,
+    ) -> Iterable[NamedUser.NamedUser]:
         github_commit = self.repo.get_commit(commit.rev)
-        yield github_commit.author.login
+        yield github_commit.author
```

Comment:
Why remove `.login` here?

Reply:
This is now yielding authors(i.e. `github.NamedUser.NamedUser` instances), so we can retrieve the `.type` attribute and check if it's a bot down the line.

I had initially implemented this as yielding a tuple of (`.login`, `.type`), but I don't have a strong preference.